Gruntfile.js

```diff
@@ -26,7 +26,7 @@ module.exports = function(grunt) {
 	  stdout: true,
 	  stderr: true,
 	  failOnError: true,
-	  npm: '--production'
+	  npm: '--development'
```

Comment:
Ah, right! This affects how the devDependencies in package.json get used, '--production' ignores them and '--development' ensures they're present. The difference also affects packaging, where with '--development' the packages listed under devDependencies get packaged with the application and with '--production' they get ignored and left out.